src/chainparams.cpp

```diff
@@ -223,16 +223,16 @@ class CTestNetParams : public CMainParams {
         consensus.hashGenesisBlock = genesis.GetHash();
         assert(consensus.hashGenesisBlock == uint256S("0x" + consensus.hashGenesisBlock.ToString()));
 
         vFixedSeeds.clear();
         vSeeds.clear();
-        vSeeds.push_back(CDNSSeedData("z.cash", "dnsseed.testnet.z.cash")); // Zcash
+        vSeeds.push_back(CDNSSeedData("tush1", "tush.cryptominingpools.org")); // Zcash
```

Comment:
need to add
`vSeeds.push_back(CDNSSeedData("node1", "testnet.myhush.network")); // madbuda`

Reply:
Added testnet.myhush.network to chainparams. I kept the tush.cryptominingpools.org there until there are more nodes. I can remove it if you would like me to.

Reply:
I will rebuild the node on tush.cryptominingpools.org this evening so it will align with the latest on the forked branch.

Reply:
Thanks, have you tried using the CPU miner? I don't see any blocks and inning crashes hushd. I would assume that the first nod will need to get difficulty up a little before others can be stable

Reply:
The CPU miner is enabled, it requires connections before it starts. When testing, I connected another node to it and mined 10 or 15 blocks. The block height is currently at 85. I do see where it failed to connect to 192.119.14.214 (testnet.myhush.network), but I believe that is just where it wasn't online. I opened up all of the ports to that VM, so try to run it again with -reindex and see if you still get the error.

Reply:
vSeeds.push_back(CDNSSeedData("node2", "testnet.madmining.club")); // madmining

another permanent testnet node

Reply:
I've updated the fork with the madmining url.